plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -294,4 +295,29 @@ public void testSelectInformationSchemaColumns()
         // Isolate this test to avoid problem described in https://github.com/trinodb/trino/issues/10846
         executeExclusively(super::testSelectInformationSchemaColumns);
     }
+
+    @Override
+    @Test
+    @Disabled
+    public void testComplexJoinPushdown()
+    {
+        // Join pushdown in SQL server produces different behavior depending
+        // on whether the plan contains an inner join node with a filter vs
+        // the filter on top of the join.
+        //
+        // The connector is producing column stats inconsistently, based on timing.
```

Comment:
The test is not supposed to use stats to determine whether join should be pushed down
https://github.com/trinodb/trino/blob/de192e466a8e8a27ff000a83fe3881b0cde01117/plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java#L872-L873

Reply:
Stats are being provided to the join reordering rule, so if stats are different, that optimization will produce different results.

Reply:
that's indeed the case.